app/src/Homepage/Homepage.css

```diff
@@ -15,8 +15,9 @@
 /* Sidebar css */
 .sidebar {
   background-color: #F0F0F0;
-  height: 100%;
+  height: 89%;
   width: 15%;
+  bottom: 0%;
```

Comment:
Is this bottom: 0% needed?

Reply:
Yes, without setting the bottom field to 0, the sidebar will begin rendering at the top of the page and will still be covered by the header. 

There may be some other way to do this (I could also move the buttons down) but my method was to shorten the sidebar and make it flush to the bottom of the page.